docs/_cli_tutorials/hydration.md

```diff
@@ -30,15 +30,15 @@ The second reason is time. Your app's integration gets 30 seconds to run its API
 
 ## How Do I use Dehydration?
 
-Check out our example files app (https://github.com/zapier/zapier-platform-example-app-files) for an example of file dehydration in action with a working Zapier demo integration. You can even initialize a Zapier app based on that repo by entering ``zapier init . --template=files`` in Terminal.
+Check out our [example files app](https://github.com/zapier/zapier-platform-example-app-files) for an example of file dehydration in action with a working Zapier demo integration. You can even initialize a Zapier app based on that repo by entering ``zapier init . --template=files`` in Terminal.
```

Comment:
The one in the zapier/zapier-platform repo is more up-to-date. Let's use that:

```suggestion
Check out our [example "files" app](https://github.com/zapier/zapier-platform/tree/master/example-apps/files) for an example of file dehydration in action with a working Zapier demo integration. You can even initialize a Zapier app based on that repo by entering ``zapier init . --template=files`` in Terminal.
```

Reply:
Thanks Chang-Hung for your feedback. I've amended the URL for the example "files" app.